File: mobile_app/lib/features/emergency/domain/entities/hospital.dart
class Hospital {
  final String id;
  final String name;
  final String address;
  final double distanceKm;
  final String phoneNumber;
  final bool emergencyAvailable;

  const Hospital({
    required this.id,
    required this.name,
    required this.address,
    required this.distanceKm,
    required this.phoneNumber,
    this.emergencyAvailable = true,
  });
}

File: backend/app/models/hospital.py
"""Hospital domain models."""

from __future__ import annotations

from dataclasses import dataclass
from typing import Optional


@dataclass
class Hospital:
    id: str
    name: str
    address: Optional[str] = None
    phone: Optional[str] = None

    latitude: Optional[float] = None
    longitude: Optional[float] = None

